modules/core/src/main/java/org/apache/ignite/internal/util/OffheapReadWriteLock.java

```diff
@@ -34,13 +35,27 @@
  * </pre>
  */
 public class OffheapReadWriteLock {
+    /** @see #IGNITE_OFFHEAP_RWLOCK_SPIN_COUNT */
+    public static final int DFLT_OFFHEAP_RWLOCK_SPIN_COUNT = 32;
+
+    /** */
+    @SystemProperty(value = "Count spin lock iterations before fallback to block", type = Long.class,
+        defaults = "" + DFLT_OFFHEAP_RWLOCK_SPIN_COUNT)
+    public static final String IGNITE_OFFHEAP_RWLOCK_SPIN_COUNT = "IGNITE_OFFHEAP_RWLOCK_SPIN_COUNT";
+
+    /** */
+    @SystemProperty(value = "OffheapReadWriteLock flag to switch between signal to writers or signal to writers or " +
```

Comment:
"The OffheapReadWriteLock flag that switches between writers and readers policy." 

Not sure if this is accurate, please revise if my understanding of the capability is wrong.